components/sup/src/manager/mod.rs

```diff
@@ -48,16 +49,16 @@ use serde;
 use serde_json;
 use time::{self, Timespec, Duration as TimeDuration};
 
-pub use manager::service::{Service, ServiceConfig, ServiceSpec, UpdateStrategy, Topology};
-use self::service::{DesiredState, StartStyle};
+pub use self::service::{Service, ServiceSpec, UpdateStrategy, Topology};
```

Comment:
Okay this isn't documented nor adhered to consistently. I'm fine leaving it as is but we need to start a style guide to communicate this.